conda/recipes/libcudf/meta.yaml

```diff
@@ -51,6 +51,7 @@ requirements:
     - librdkafka {{ librdkafka_version }}
     - fmt {{ fmt_version }}
     - spdlog {{ spdlog_version }}
+    - gtest==1.10.0.*
```

Comment:
Use the existing variable.
https://github.com/rapidsai/cudf/blob/9ceecb1346bbdf79a3ba104a3e456cc07ce92762/conda/recipes/libcudf/conda_build_config.yaml#L16

Reply:
Ah I forgot that we had that. Thanks. Actually I think we can move both gtest and gmock out of the run requirements for libcudf-tests and into the run requirements for libcudf itself. libcudftestutil is part of libcudf, so even if someone isn't running our test suite they still need gtest if they want to use the test utils that are part of libcudf.